NotationV0/src/install.ts

```diff
@@ -0,0 +1,43 @@
+import * as path from 'path';
+import * as taskLib from 'azure-pipelines-task-lib/task';
+
+import { getDownloadInfo, installFromURL } from './lib/install';
+
+import { NOTATION_VERSION_FILE } from './lib/constants';
+
+export async function install(): Promise<void> {
+    // the notation download URL
+    let version: string
+    let downloadURL: string
+    let checksum: string
+
+    if (taskLib.getBoolInput('isCustomVersion', false)) {
+        // for custom version, download the notation from specified URL
+        version = "1.0.0-UNRELEASED"
```

Comment:
`version` should not be set for custom version. Instead, we should log differently, explicity stating it is a custom version.

Reply:
Removed it.